src/experimental/safe_html_transform/transforms/exp_safe_html.py

```diff
@@ -293,24 +293,22 @@ def convert(self, orig, data, **kwargs):
         else:
             # append html tag to create a dummy parent for the tree
             html = "<html>%s</html>" % orig
+            parser = etree.HTMLParser()
+            tree = etree.parse(StringIO(html), parser)
+            for element in tree.getiterator():
+                if(element.tag == 'h3' or element.tag == 'h4' or element.tag == 'h5' or element.tag == 'h6' or element.tag == 'div'):
+                    element.tag = 'p'
+                if(element.tag == "html" or element.tag == "body" or element.tag == "script"):
+                    etree.strip_tags(tree, element.tag)
```

Comment:
Aha, didn't realise that's what kill_tags does. Whether we want to remove the content depends on the tag. For ``<body>`` then yes, we want to keep the content so strip_tags is good. But for ``<script>``, we want to throw away the javascript content within too, so kill_tags is the right choice there instead of strip tags.